src/NzbDrone.Core/Indexers/TorrentRssIndexer/TorrentRssIndexerParserSettings.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NzbDrone.Core.Indexers.TorrentRssIndexer
+{
+    public class TorrentRssIndexerParserSettings
```

Comment:
I don't think settings (as in data like this) should be provided to the generic indexer, it should follow other indexer where you tell it what type/instance of parser it should use. the parser could accept settings, but not the indexer client itself.

this makes indexer too involved in parsing, and we don't like overly involved components in sonnar ;) we try to keep thing as specialized and delegate unrelated tasks.